doc/ikfast/ikfast_tutorial.rst

```diff
@@ -1,64 +1,44 @@
 IKFast Kinematics Solver
 ========================
 
-.. image:: openrave_panda.png
-   :width: 700px
```

Comment:
The image is not expressive for this tutorial.

Reply:
During our last refactor, we had a goal to add images to every tutorial to make them feel more friendly and less boring.

This picture @mlautman made using OpenRave, which is very relevant to this tutorial. please restore it.

Reply:
I know that this image is from OpenRave. However, I'm not convinced yet that it is useful in any fashion for this tutorial.
For me, it's just distracting. Following the given tutorial, the user will never see a similar image.

Reply:
This isn't a technical question, its a user-interface question. Many users of the tutorials might be intimidated by the complexity of moveit, pictures soften that blow in some kind of psychological way. If you have a better pic, please suggest. Otherwise, let's keep this work @mlautman put into the tutorials

Reply:
I bow to the majority and will restore the image. However, I'm still not convinced.